tests/test-backend-ops.cpp

```diff
@@ -476,6 +476,9 @@ struct test_unary : public test_case {
     ggml_tensor * build_graph(ggml_context * ctx) override {
         ggml_tensor * in = ggml_new_tensor(ctx, type, 4, ne.data());
         ggml_tensor * out = ggml_unary(ctx, in, op);
+        if(op == GGML_UNARY_OP_LEAKY_RELU) {
+            ((int32_t *)(out->op_params))[1] = (int32_t)(0.1f /* negative slope*/ * 100.0f);
+        }
```

Comment:
Maybe it would be better to make `ggml_leaky_relu` a standalone op instead of using `GGML_UNARY`.